assets/css/style.css

```diff
@@ -0,0 +1,197 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.header h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.header h1 .seo {
+  color: #d9dcd6;
+}
+/* decendent conmbinator */
```

Comment:
Comments in your CSS file should help us understand which part of the html code is being styled (as it is not always apparent just by looking at the selectors). Please try and add some further descriptive comments through this file.

Reply:
im not sure what comments to add? ideas?